packages/api/src/context.ts

```diff
@@ -46,6 +46,26 @@ import {Client, Issuer} from 'openid-client'
 import {MailContext, MailContextOptions} from './mails/mailContext'
 import {User} from './db/user'
 import {ChallengeProvider} from './challenges/challengeProvider'
+import NodeCache from 'node-cache'
+
+/**
+ * Peered article cache configuration and setup
+ */
+const peerCache = new NodeCache({
+  stdTTL: 1800,
+  checkperiod: 60,
+  deleteOnExpire: false,
+  useClones: false
+})
+peerCache.on('expired', async function (key: string, value: peerCacheValue) {
+  // Refresh cache only if last use of cached entry is less than 24h ago
+  if (value.queryParams.lastQueried > new Date().getTime() - 24 * 60 * 60 * 1000) {
```

Comment:
putting code such as 24 * 60 ... in a meaning full variable:

const ONE_DAY_IN_MS = 24 * 60 * 60 * 1000

Reply:
I would probably wait for 2 days.

Reply:
@michael-scheurer why would you wait for 2 days? I don't think that's necessary. Since an article really would need to not be accessed for 24h. And that only in the case an other node is offline for more than 24h. This should not be very likely ;-)

Reply:
Fixed